tests/unit/Makefile

```diff
@@ -29,4 +29,4 @@ test_mem_map_ptr: test_mem_map_ptr.c
 test_mem_high: test_mem_high.c
 
 ${ALL_TESTS}:
-	gcc ${CFLAGS} -o $@ $^
+	clang ${CFLAGS} -o $@ $^
```

Comment:
why do you change gcc to clang?

btw, should we change "gcc" to $CC so this works with all compilers?